### PR TITLE
feat: inventory script: parse and support location data.

### PR DESCRIPTION
**Pull request by merlin-northern**
Changelog: Title
Ticket: MEN-5915

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please baby

**Review by alfrunes — COMMENTED** [review]
I'm not sure what the requirements of this task is. Should the data be treated as a single array/tuple (point) attribute? Or do we store it as two separate attributes as we do in this PR?

**Comment by merlin-northern** [comment]
> I'm not sure what the requirements of this task is. Should the data be treated as a single array/tuple (point) attribute? Or do we store it as two separate attributes as we do in this PR?

the reqs were clear, about two of them, thats why i did it like that. you prefer without split?

File: support/mender-inventory-geo [hunk 1]
```diff
@@ -34,6 +34,8 @@ ATTR_NAME_IP="geo-ip"
 ATTR_NAME_CITY="geo-city"
 ATTR_NAME_COUNTRY="geo-country"
 ATTR_NAME_TIMEZONE="geo-timezone"
+ATTR_NAME_LAT="geo-lat"
+ATTR_NAME_LON="geo-lon" # loc=latitude,longitude
```
Comment (kjaskiewiczz) [inline support/mender-inventory-geo:38]: strange place for the _# loc=latitude,longitude_ comment

**Review by kjaskiewiczz — APPROVED** [review]
single attribute would be enough, but two separate attributes looks less error prone;
no strong opinion, so I'm fine with current implementation

File: support/mender-inventory-geo (hunk 1 above)
Comment (merlin-northern) [reply support/mender-inventory-geo]: fixed, I hope you like it.

Comment (merlin-northern) [reply support/mender-inventory-geo]: Edit: I know you like it :) because we discussed privately :)

**Comment by merlin-northern** [comment]
I am not re-running the full pipeline as the change was only moving the comment:
```diff
+ # the following two attributes denote latitude (geo-lat) and longitude (geo-lon)
+ # as returned by the ipinfo.io in the loc field (loc=latitude,longitude)
ATTR_NAME_LAT="geo-lat"
- ATTR_NAME_LON="geo-lon" # loc=latitude,longitude
+ ATTR_NAME_LON="geo-lon"
```

**Pull request merged**